mongo/collection.go

```diff
@@ -1188,8 +1295,17 @@ func (coll *Collection) findAndModify(ctx context.Context, op *operation.FindAnd
 	return &SingleResult{rdr: bson.Raw(op.Result().Value), reg: coll.registry}
 }
 
-// FindOneAndDelete find a single document and deletes it, returning the
-// original in result.
+// FindOneAndDelete executes a findAndModify operation
```

Comment:
I don't mind the link to the command reference, but I wouldn't lead with it.  Instead, say what the effect is, and put the link at the bottom.

Reply:
Changed for each method in client/database/collection.